postem/postem-app/pom.xml

```diff
@@ -7,124 +7,110 @@
     <version>22-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <name>Sakai Postem Tool</name>
-  <groupId>org.sakaiproject</groupId>
+  
+  <name>sakai-postem-tool</name>
   <artifactId>sakai-postem-tool</artifactId>
   <packaging>war</packaging>
 
-    <!-- profiles added for WAS support -->
-    <profiles>
-        <profile>
-            <id>full</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-            <dependencies>
-                <dependency>
-                  <groupId>org.sakaiproject.jsf</groupId>
-                  <artifactId>myfaces-tool</artifactId>
-                  <type>pom</type>
-		  <exclusions>
-		    <!-- exclude this to avoid the collision btw RI and tomahawk -->
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-api</artifactId>
-		    </exclusion>
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-impl</artifactId>
-		    </exclusion>
-		  </exclusions>
-                </dependency>
-            </dependencies>
-        </profile>
-        <profile>
-            <!-- exclude xerces xml parser apis for WAS -->
-            <id>was</id>
-            <dependencies>
-                <dependency>
-                  <groupId>org.sakaiproject.jsf</groupId>
-                  <artifactId>myfaces-tool</artifactId>
-                  <type>pom</type>
-		  <exclusions>
-		    <!-- exclude this to avoid the collision btw RI and tomahawk -->
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-api</artifactId>
-		    </exclusion>
-		    <exclusion>
-		      <groupId>javax.faces</groupId>
-		      <artifactId>jsf-impl</artifactId>
-		    </exclusion>
-		  </exclusions>
-                </dependency>
-            </dependencies>
-        </profile>
-    </profiles>
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
+        <thymeleaf.version>3.0.11.RELEASE</thymeleaf.version>
```

Comment:
Hi.
I've moved Thymeleaf dependencies from postem to master/pom.xml.

Reply:
After consulting Miguel about auto groups (datemanager and site-group-manager) he told me that it would be necessary to open another Jira with the request and we treated it separately.

Reply:
One more thing.

I have reversed this change because datemanager and site-group-manager did not start with the change of location of dependencies.

We wait for the tour ticket to change these Thymeleaf dependencies.